function.hpp

```diff
@@ -484,7 +484,7 @@ struct storage_t<signature<ReturnType(Args...)>, Qualifier, Config>
 
     vtable_ptr_t _vtable;
 
-    void* _impl;
+    void* _impl = nullptr;
```

Comment:
Setting the implementation to null is unneccessary since the constructor calls `tidy()`